ql/time/calendars/poland.cpp

```diff
@@ -18,16 +18,28 @@
 */
 
 #include <ql/time/calendars/poland.hpp>
+#include <ql/errors.hpp>
 
 namespace QuantLib {
 
-    Poland::Poland() {
+    Poland::Poland(Poland::Market market) {
         // all calendar instances share the same implementation instance
-        static ext::shared_ptr<Calendar::Impl> impl(new Poland::Impl);
-        impl_ = impl;
+        // static ext::shared_ptr<Calendar::Impl> impl(new Poland::Impl);
```

Comment:
No need to keep the old one commented, you can remove it

Reply:
Yes, my oversight, already corrected